models/dave_robot_models/description/rexrov/model.sdf

```diff
@@ -584,5 +584,22 @@
       <joint_name>thruster8_joint</joint_name>
     </plugin>
 
+    <!-- Sea Pressure Sensor Plugin -->
+    <plugin
+      filename="sea_pressure_sensor"
+      name=" dave_gz_sensor_plugins::SubseaPressureSensorPlugin">
+      <namespace>rexrov</namespace>
+      <!-- <link_name>base_link</link_name> -->
+      <topic>sea_pressure</topic>
+      <!-- <mass>0.015</mass> -->
```

Comment:
Is this `mass` needed anytime later? if not should be removed

Reply:
Yess, we also have a pressure sensor model, I will add it in another PR in the future.
For now I have just added it as we might need it later, Shall I remove it for this PR and add it once the Model is Migrated ?